h5py/tests/test_group.py

```diff
@@ -19,7 +19,7 @@
 
 import numpy as np
 import os
-import os.path
```

Comment:
to be clear I'm removing this import because it's no-op (with `os` already imported). I'm not replacing it with pathlib